src/NUnitFramework/framework/Api/FrameworkController.cs

```diff
@@ -174,7 +177,15 @@ private void Initialize(string assemblyPath, IDictionary settings)
         /// <summary>
         /// Gets a dictionary of settings for the FrameworkController
         /// </summary>
-        public IDictionary Settings { get; private set; }
+        public IDictionary Settings
+        {
+            get { return (IDictionary)InternalSettings; }
+        }
+
+        /// <summary>
+        /// Gets a dictionary of settings for the FrameworkController
+        /// </summary>
+        internal IDictionary<string, object> InternalSettings { get; private set; }
```

Comment:
I am not sure this is even necessary.  All the other classes that use Settings keep their copy internal.

It is only because this is released that this seems necessary.  If we ask, and no one is using the public Settings property, it can be removed without any problem, and the InternalSettings can then be renamed to Settings.